src/components/Tabber/Tabber.tsx

```diff
@@ -58,7 +57,13 @@ function Tabber() {
           />
         </div>
         <div className={scss.previewContainer}>
-          <preview.previewComponent text={`${preview.name} Placeholder`} />
+          {preview?.previewComponent ? (
+            <preview.previewComponent
+              customData={{}}
+              medias={[] as File[]}
+              text={`${preview.name} Placeholder`}
+            />
+          ) : null}
```

Comment:
Acho que podemos falar sobre esse bloco de código

Reply:
Fiz a correção mas infelizmente o eslint acusou: `Potential leaked value that might cause unintentionally rendered values or rendering crasheseslint[react/jsx-no-leaked-render](https://github.com/jsx-eslint/eslint-plugin-react/tree/master/docs/rules/jsx-no-leaked-render.md)`

tentei usar o operador de negação dupla, assim:

```js
{!!(preview?.previewComponent) && (
    <preview.previewComponent
      customData={{}}
      medias={[] as File[]}
      text={`${preview.name} Placeholder`}
    />
 )}
```

Mesmo assim continuou o erro.

Reply:
Pode remover essa rule do lint